netbox/utilities/forms/widgets.py

```diff
@@ -310,3 +313,39 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.attrs['class'] = 'time-picker'
         self.attrs['placeholder'] = 'hh:mm:ss'
+
+
+class MemoryWidget(forms.MultiWidget):
+    """
+    Memory Widget.
+    """
+    def __init__(self, attrs=None):
+        widgets = (
+            forms.NumberInput(),
+            StaticSelect(choices=add_blank_choice(MemoryUnitChoices.MEMORY_CHOICES))
+        )
+        super(MemoryWidget, self).__init__(widgets, attrs)
+
+    def decompress(self, value):
+        if value:
+            return [value, MemoryUnitChoices.UNIT_MB]
+        else:
+            return ['', '']
+
+
+class DiskWidget(forms.MultiWidget):
```

Comment:
Same as above; we should only need one widget class.

Reply:
Done.